query.go

```diff
@@ -57,7 +59,9 @@ func NewDateRangeQuery(start, end time.Time) *query.DateRangeQuery {
 // NewDateRangeInclusiveQuery creates a new Query for ranges
 // of date values.
 // Date strings are parsed using the DateTimeParser configured in the
-//  top-level config.QueryDateTimeParser
```

Comment:
ditto

Reply:
This is for the benefit of [gopls](https://pkg.go.dev/golang.org/x/tools/gopls)